content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Introducing MOSN: An Alternative Data Plane"
+subtitle: "A Cloud Native Proxy for Edge or Service Mesh"
+description: "An alternative sidecar proxy for Istio."
+publishdate: 2020-07-17
+attribution: "Wang Fakang(mosn.io)"
+keywords: [mosn,sidecar,proxy]
+---
+
+Thanks to the efforts of the MOSN community, make MOSN has completed the most of adaptation for Istio. MOSN [v0.14.0 release](https://github.com/mosn/mosn/releases/tag/v0.14.0) is now compatible with Istio 1.5.x, and it has gone through the Bookinfo sample as the data plane of Istio.
+
+## Background
+
+In the service mesh world, using Istio as the control plane has become the mainstream. Istio provides dynamic configuration of routes, service discovery, etc. to data plane proxies via the xDS protocol. The proxy can conveniently serve as Istio's data by simply interfacing with Envoy's xDS protocol. Istio's integration of third-party data planes can be implemented in three steps, as follows.
+
+- Implement xDS protocols to fulfill the capabilities for data plane related services.
+- Build `proxyv2` images using Istio's script and set the relevant `SIDCAR` and other parameters.
+- Specify a specific data plane via the `istioctl` tool and set the proxy-related configuration.
+
+## What is MOSN?
+
+MOSN is a network proxy written in GoLang. It can be used as a cloud-native network data plane, providing services with the following proxy functions: multi-protocol, modular, intelligent, and secure. MOSN is the short name of Modular Open Smart Network (proxy). MOSN can be integrated with any service mesh which supports xDS API. MOSN can also be used as an independent Layer 4 or Layer 7 load balancer, API Gateway, cloud-native Ingress, etc.
+
+## Architecture
+
+MOSN follows the OSI (Open Systems Interconnection), it has four layers, NET/IO, Protocol, Stream, and Proxy, as shown in the following figure.
```

Comment:
I don't see those layers written down in https://en.wikipedia.org/wiki/OSI_model.

Are you trying to say "Just like TCP and OSI, we have used a layered model"?

Reply:
Yes.

Reply:
I found it complicated; you might want to say "MOSN has a layered architecture; NET/IO.. " etc

Reply:
Ok, it has been updated. Thank you.